crates/apis/README.md

```diff
@@ -27,16 +27,6 @@ If you want to customize the runtime or the APIs, you can use the `Runtime::new_
 * `random` - Overrides the implementation of `Math.random` to one that seeds the RNG on first call to `Math.random`. This is helpful to enable when using Wizer to snapshot a Javy Runtime so that the output of `Math.random` relies on the WASI context used at runtime and not the WASI context used when Wizening. Enabling this feature will increase the size of the Wasm module that includes the Javy Runtime and will introduce an additional hostcall invocation when `Math.random` is invoked for the first time.
 * `stream_io` - Registers implementations of `Javy.IO.readSync` and `Javy.IO.writeSync`.
 
-## Building a project using this crate
```

Comment:
Might be good to leave a note _somewhere_ that you can point to a custom WASI SDK using the env variable.